### PR TITLE
docs: add Google Play badge to README.md and remove contributors

### PR DESCRIPTION
**Pull request by mipro98**
I improved our README a bit: I added a Google Play badge and removed the Contributors section for now since it is visible anyways on the sidebar of the repository in an equal fashion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,25 +11,32 @@ Your personal practice companion
 </p>
 
 <br/>
+
+<p align="center">
+<a href='https://play.google.com/store/apps/details?id=app.musikus&hl=de&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'>
```
Comment (matthiasemde) [inline README.md:16]: I don't think, we should use the GooglePlay Badge while the link only works for our alpha testers.

File: README.md [hunk 2]
```diff
@@ -11,25 +11,32 @@ Your personal practice companion
...
 <br/>
 
+
 ## 🧪 Musikus is in alpha
 
 ### Join
-If you want to join our group of testers, write us an email to [alpha@musikus.app](mailto:alpha@musikus.app) using your Gmail-account and we will add you to the list.
+
+If you want to join our group of testers, write us an email to [alpha@musikus.app](mailto:alpha@musikus.app) using your Google account and we will add you to the list. If you are amongst the list of testers, the badge above will bring you to the Play Store listing.
+Alternatively, you can always download the latest release [from GitHub](https://github.com/matthiasemde/musikus-android/releases/latest).
```
Comment (matthiasemde) [inline README.md:34]: People _can_ download the latest release from GitHub, but then they are not really joining the alpha. The alpha is happening on GooglePlay.

File: README.md [hunk 3]
```diff
@@ -11,25 +11,32 @@ Your personal practice companion
...
 
 ### Feedback
-If you have any feedback, bug reports or feature requests, please fill out [this](https://docs.google.com/forms/d/e/1FAIpQLSfw1rTslL7b3Yp_Lv_E65zketfj7G6h5VzOkVdxjcQZDC5CqA/viewform) form or use the [Issue Tracker](https://github.com/matthiasemde/musikus-android/issues/new).
 
-## 👨‍💻 Contributors
-<a href="https://github.com/matthiasemde/musikus-android/graphs/contributors">
-  <img width="140" src="https://contrib.rocks/image?repo=matthiasemde/musikus-android" alt="https://github.com/matthiasemde/musikus-android/graphs/contributors"/>
-</a>
```
Comment (matthiasemde) [inline README.md:32]: I know it's a gimmick, but I really like the contributors page.

**Review by matthiasemde — COMMENTED** [review]
I would decline this PR. But we can of course talk about it if you want.

File: README.md (hunk 1 above)
Comment (mipro98) [reply README.md:16]: Do you think it confuses people who don't read the description and wonder why the link is not working 😅?

File: README.md (hunk 2 above)
Comment (mipro98) [reply README.md:34]: Okay, fair point. But I would like to encourage people to test the app even though they might not want to be an "official alpha tester".
It's just about mentioning that our releases are also uploaded to GitHub.

Maybe we can rephrase it to something like "Pre-built release apks can be found on the release page".

**Comment by mipro98** [comment]
Since the CI badge showed "failing" because of a recent CI run in an intermediate branch in a PR, I added `?branch=main` as described [here](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-branch-parameter) but it indicated "no status". @matthiasemde do you have an idea why?

**Comment by matthiasemde** [comment]
> Since the CI badge showed "failing" because of a recent CI run in an intermediate branch in a PR, I added `?branch=main` as described [here](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-branch-parameter) but it indicated "no status". @matthiasemde do you have an idea why?
My guess is, that the CI has never actually run for `branch=main`. It only ever runs for branches for which a PR is opened.
That being said, I'm not really sure what the desired behavior is in our case.

**Review by matthiasemde — APPROVED** [review]

**Comment by mipro98** [comment]
For screenshots, I'd take these:

Screenshot 1
Screenshot 2
Screenshot 3
Screenshot 4
Screenshot 5
Screenshot 6
Screenshot 78afe)

**Comment by mipro98** [comment]
We may also include a IzzyOnDroid badge: https://gitlab.com/IzzyOnDroid/repo#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it

badge

Also, Izzy provides  shield we could add:

shield

File: README.md [hunk 4]
```diff
@@ -11,20 +11,41 @@ Your personal practice companion
...
-If you want to join our group of testers, write us an email to [alpha@musikus.app](mailto:alpha@musikus.app) using your Gmail-account and we will add you to the list.
+<br>
 
-### Feedback
-If you have any feedback, bug reports or feature requests, please fill out [this](https://docs.google.com/forms/d/e/1FAIpQLSfw1rTslL7b3Yp_Lv_E65zketfj7G6h5VzOkVdxjcQZDC5CqA/viewform) form or use the [Issue Tracker](https://github.com/matthiasemde/musikus-android/issues/new).
+</div>
+
+## ✨ Features
+
+**Musikus is an app designed for musicians to support them during practicing with various handy features and tools.**
```
Comment (matthiasemde) [inline README.md:38]:
```suggestion
**Musikus is the ultimate practice companion for musicians, designed to track progress, boost motivation, and streamline practice sessions. With all the tools you need in one app, Musikus makes every practice seamless and effective.**
```

File: README.md [hunk 5]
```diff
@@ -11,20 +11,41 @@ Your personal practice companion
...
+
+* 📅 **Session Overview**: Review your past practice sessions at a glance.
+* 🎵 **Metronome**: Keep perfect time with our precise metronome.
```
Comment (matthiasemde) [inline README.md:41]:
```suggestion
* 🎵 **Metronome**: Stay perfectly in time with our precise metronome.
```

File: README.md [hunk 6]
```diff
@@ -11,20 +11,41 @@ Your personal practice companion
...
+* 🎤 **Recorder**: Capture your practice sessions and listen back to track your improvement.
+* 🎯 **Goals**: Set and achieve daily, weekly, and monthly goals to stay motivated and on track.
```
Comment (matthiasemde) [inline README.md:43]:
```suggestion
* 🎯 **Goals**: Set and achieve daily, weekly, and monthly goals to stay motivated.
```
Three "and"s in a row sound a bit funny imo.

File: README.md [hunk 7]
```diff
@@ -11,20 +11,41 @@ Your personal practice companion
...
+* 📊 **Statistics**: Gain insights into your practice habits with detailed statistics.
+
+## 🧪 Musikus is in beta
```
Comment (matthiasemde) [inline README.md:46]:
```suggestion
## 🧪 Musikus is in Open Beta
```

File: README.md [hunk 8]
```diff
@@ -11,20 +11,41 @@ Your personal practice companion
 </p>
 
 <br/>
+
+<p align="center">
+<a href='https://play.google.com/store/apps/details?id=app.musikus'>
```
Comment (matthiasemde) [inline README.md:16]: I think I would move the Google Play, F-Droid and whatnot badges further down. The GitHub badges are such an active element on the readme page that they deserve the prime real-estate on top. However, I guess you might be concerned that might not realize, that Musikus is an Android app... which is true.. but still

**Review by matthiasemde — CHANGES_REQUESTED** [review]

**Comment by matthiasemde** [comment]
Concerning the screenshots I have only two (and a half) remarks. I would change some of the item and folder names to something more realistic. The sessions list screenshot contains two sessions which both have duplicate sections. And I personally like the bar chart more than the pie chart. Especially if you deactivate some of the items until its only 3-4 colors 👌. Otherwise, screenshots are looking really nice. The app looks so nice. From the UI design to the Colors 😍

File: README.md (hunk 4 above)
Comment (mipro98) [reply README.md:38]: Nice text, maybe for the store but for a (technical) github readme IMO I find it a bit too "sensational".

Comment (matthiasemde) [reply README.md:38]: Maybe, however the previous text didn't even mention time tracking... Also we could simply commit to keeping this description and the one in the google store in sync which makes it easier for us

**Comment by matthiasemde** [comment]
Something like this?

**Comment by mipro98** [comment]
I would also like add a "get it on "GitHub" badge to promote our own release channel we've set up. Tehre is a badge arount ([Link](https://raw.githubusercontent.com/NeoApplications/Neo-Backup/034b226cea5c1b30eb4f6a6f313e4dadcbb0ece4/badge_github.png)) however it seems there is no "official" source for it since all repos I found which use it include it either from a foreign source or include it in `/assets`

File: README.md [hunk 9]
```diff
@@ -11,20 +11,48 @@ Your personal practice companion
...
...5JiOZkFxUFZo74mgKTqaKXu0+2HqVwkja3BH9kFu361JwcHTfPJD4mdfe8ULAdVRyGlJAcVFfg+CQOozZ4XrJ85+JgwBsVXIGriQw5Tp4ZScezd8JiWnBupru30qwJZa+ZAjmWlC8fUZM4qB6kPnLNSPLMWqQQ5ZQ5aOzs1HmamBaQHzFs6y+qAmJCTE8f9/QgKSBg4DJPWc6zVDQkIC09JkZSPD38kukpExFpT4z67uYI/QwCOOCCK/izvu5CWl6AcEWMnKWml7LWbKZfH9/99UkknQHhGsynDz+65eWXv3/JmJrq5eXienVlRUfH/z8VvCf45soKQIH1yDEQsszrp6gwq9C73T87xcXadKl5TkFev4A/2tygmSBqYXqAYJmK+ZuoJydDR1vP09DA0NOy2kpdML81+U/heCpH1JU3jig7lJ5nKOT4i/t6ZHkqGzs4lJmIVHfrj+JR4HqLQSD0yDkCNEpGNn5ix9D03/eJdElTZdKV2TpNOhkwt8YUlNUgimgV0dLMBvf1gz1MolPd5FRcVNSkpDQ8owJeBCDyIhrIDnOD5QcuIU+3/2QKSs9laQ+noNLS0zLWdtqyP7mBAFAw88TwsJgMuJYweBGjYngtWbmeuZOW+bvNQToUFOAlFqOBk4Ov3/L7Z60/aN0p1tUhpa5nqWlub7C3p2I9QzyAghlUvczOz/1fhzPT3XSIfpSmmYAdVbmm1gV0dSz8DSilpUQsqCddIWIA3meuZaJqdMJZEzl6gRqgZIWZAxUdoizERXN8yi5MltcZTChzMaRQM3JNUWHS8rL/+yaPGvMmvr5ywoGoxtkDWwQ+Pb89ycBeWfGSJeL/la+RS1eOPnRtbQKgMRjZg+t8x6PkP273nWQAoFOPAgaeAThKXAmXMrK39Kmr5fsuBlBqoXfJGLe3VbmHjG9Mczi9T//3h7vygXtcDlQtJg44iQiIjIBRbGPO7gghPJy0ZIxT2HOLIUgwxQzsgYrUR350HSIMaJLidhgKY+mw+pflBDrX8E7OGBjPCAPc76gQFSTqAIiYrb/8dRP4CyosJ/rmwU5XIxHMilt4QBJwsSkBMClxOQULBlkRRwEONmR2kJcDGjADX2/+xO8r5iqjExqmLyrWpcPFRta1BfAwCtyN3XpuJ4RgAAAABJRU5ErkJggg==" alt="nighlty-release"></a>
```
Comment (matthiasemde) [inline README.md:20]: do we need both the shield **and** the "Get it on Izzidroid" Image? I think i'd prefer the image over the shield...

File: README.md (hunk 4 above)
Comment (matthiasemde) [reply README.md:45]: I think we could replace the "ultimate practice companion" with sth less sensational like: "perfect practice companion" but I would prefer both versions over the current text 😅

**Comment by matthiasemde** [comment]
> I would also like add a "get it on "GitHub" badge to promote our own release channel we've set up. There is a badge around Badge ([Link](https://raw.githubusercontent.com/NeoApplications/Neo-Backup/034b226cea5c1b30eb4f6a6f313e4dadcbb0ece4/badge_github.png)) however it seems there is no "official" source for it since all repos I found which use it include it either from a foreign source or include it in `/assets`

I don't now if we want to include a random link to a foreign source as it might break without us noticing.. checking it into our repo feels a bit weird but would probably be the cleanest solution. What about the screenshots... would you put them into '/assets' as well?
We could also just drag and drop them in the browser and have GitHub store them for us 🙃

**Review by matthiasemde — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (mipro98) [reply README.md:20]: It's only for versioning. If we include the github release version shields.io badge, we should also include the IzzyOnDroid badge to show their version. - Or none
